account.go

```diff
@@ -1,22 +1,260 @@
 package stripe
 
-import "encoding/json"
+import (
+	"encoding/json"
+	"fmt"
+	"net/url"
+	"strconv"
+)
+
+type LegalEntityType string
+
+type IdentityVerificationStatus string
+
+type Interval string
+
+const (
+	Individual  LegalEntityType = "individual"
```

Comment:
OK, I grabbed the list from https://stripe.com/docs/marketplaces/accounts#legal-entity